graph/layout/plotter_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package layout
+
+import (
+	"fmt"
+	"image/color"
+	"math"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
+)
+
+const radius = vg.Length(15)
+
+// render implements the plot.Plotter interface for graphs.
```

Comment:
Do you plan to export this in some form later?

Reply:
Yes, but it needs to have more user control over the rendering options. That needs a fair amount of thought.